### PR TITLE
feat: add file checksum to file name for caching purposes [KMAPS-385]

### PR DESCRIPTION
**Pull request by nelsonmestevao**

### CODE REVIEW

**Review by djcouto — APPROVED** [review]
🚀

File: packages/angular-ui-components/README.md [hunk 1]
```diff
@@ -109,9 +109,27 @@ In addition, you can customize the theme by changing some CSS properties.
 }
 ```
 
+## Caching
+
+By default, the `KvIcon` and `KvIllustration` components require an SVG file with all the `kv-icons` available. The default file is `svg-symbols.svg` which is provided after installing this dependency. For caching purposes, it is also provided a `symbols.${checksum}.svg` file after installation. If you are caching those SVGs in your project you should provide the latter to the library configuration in your `app.component.ts`.
+
+```tsx
```
Comment (pedrorebelocosta) [inline packages/angular-ui-components/README.md:116]: minor detail but use typescript md as the example is for angular which doesn't use JSX.

File: packages/angular-ui-components/.scripts/copy-assets.js [hunk 2]
```diff
@@ -2,15 +2,52 @@ console.log(`-------------------------------------------------
...
+		});
+
+		input.on("close", function () {
+			resolve(hash.digest("hex"));
+		});
+	});
+}
+
+getFileChecksum(SYMBOLS_FILE_PATH).then((hash) => {
+	const timestamp = new Date().toISOString().split(".")[0];
```
Comment (djcouto) [inline packages/angular-ui-components/.scripts/copy-assets.js:31]: 🤭
```suggestion
	const [timestamp] = new Date().toISOString().split(".");
```

**Review by pedrorebelocosta — APPROVED** [review]

File: packages/react-ui-components/.scripts/copy-icons.js [hunk 3]
```diff
@@ -2,15 +2,50 @@ console.log(`-------------------------------------------------
...
+		});
+
+		input.on('close', function () {
+			resolve(hash.digest('hex'));
+		});
+	});
+}
+
+getFileChecksum(SYMBOLS_FILE_PATH).then(hash => {
+	const timestamp = new Date().toISOString().split('.')[0];
```
Comment (djcouto) [inline packages/react-ui-components/.scripts/copy-icons.js:31]: Same 😄

**Review by djcouto — APPROVED** [review]

**Comment by pedrorebelocosta** [comment]
Great improvement :D

File: packages/angular-ui-components/.scripts/copy-assets.js (hunk 2 above)
Comment (nelsonmestevao) [reply packages/angular-ui-components/.scripts/copy-assets.js]: 🧞

File: packages/angular-ui-components/README.md (hunk 1 above)
Comment (nelsonmestevao) [reply packages/angular-ui-components/README.md]: Done :muscle:

**Review by miguelpinto98 — APPROVED** [review]
Very nice workaround this problem! It's a solution that increase or performance. 👏

**Pull request merged**